Controllers/SongController.cs

```diff
@@ -85,6 +90,51 @@ public async Task<ActionResult<Song>> RefreshHash(string hash)
             return song;
         }
 
+        [HttpGet("~/map/modinterface/{hash}")]
+        public async Task<ActionResult<List<DiffModResponse>>> GetModSongInfos(string hash)
+        {
+            ICollection<DifficultyDescription>? diffs = await _context.Songs
+                .Where(el => el.Hash == hash)
+                .Select(song => song.Difficulties)
+                .FirstOrDefaultAsync();
+
+            if (diffs is null)
+            {
+                return NotFound();
+            }
+
+            List<DiffModResponse> result = new();
+
+            foreach(DifficultyDescription? diff in diffs)
+            {
+                if(diff is not null)
+                {
+                    result.Add(new()
+                    {
+                        DifficultyName = diff.DifficultyName,
+                        ModeName = diff.ModeName,
+                        Stars = diff.Stars,
+                        Nominated = diff.Nominated,
```

Comment:
`Ranked`, `Nominated` and `Qualified` were replaced by the `Status`. Bools are still here for backward compatibility

Reply:
Changed this to Status